src/plugins/intel_cpu/src/nodes/interpolate.cpp

```diff
@@ -361,11 +375,18 @@ struct jit_uni_interpolate_kernel_f32 : public jit_uni_interpolate_kernel, publi
                 mov(reg_index_offset, dword[reg_index]);
                 add(reg_src_aux, reg_index_offset);
 
-                load(reg_src_aux, vmm_val, scalar_step);
-                if (attr_.post_ops_.len() != 0)
-                    apply_post_ops(jcp_.dst_prc, 1);
-                store(vmm_val, reg_dst, scalar_step);
-
+                if (is_i8) {
+                    load(reg_src_aux, vmm_val, scalar_step, 0, Precision::I8);
+                    if (attr_.post_ops_.len() != 0) {
+                        IE_THROW() << "Interpolate I8 doesn't support fusing";
+                    }
+                    store(vmm_val, reg_dst, scalar_step, 0, Precision::I8);
+                } else {
+                    load(reg_src_aux, vmm_val, scalar_step);
+                    if (attr_.post_ops_.len() != 0)
+                        apply_post_ops(jcp_.dst_prc, 1);
+                    store(vmm_val, reg_dst, scalar_step);
+                }
```

Comment:
for all the tails, we can handle it in one vector process, without loop. Gather all the tails to vector register + post_ops + store with store_number parameter. To support this, we need emu_gather() with gather_num parameter.

Reply:
Done